docs/governance/overview.md

```diff
@@ -1,26 +1,26 @@
 ---
 id: overview
 slug: /governance/
-title: Participate in Governance 
+title: Participate in Governance
 sidebar_label: Overview
-description: Governance is an essential topic that includes Cardano Improvement Proposals (CIP), project funding, voting, and network parameters.
+description: Governance is an essential topic that includes the on-chain governance framework, Cardano Improvement Proposals (CIP), Cardano Problem Statements (CPS), Project Catalyst, and Network Parameters.
 image: /img/og/og-developer-portal.png
 ---
 
 ![Cardano Governance](../../static/img/card-governance-title.svg)
 
-Cardano was designed with the intent of long-term sustainability in mind. The goal was to build a system that could govern and fund itself without relying on donations. Cardano includes a built-in treasury system that receives revenue via transaction fees and treasury taxes on a regular basis. This is explained in detail in [treasuries with Bingsheng Zhang](https://www.youtube.com/watch?v=Hyh3h_yX-S0). 
+Cardano has been meticulously designed to ensure sustainability and longevity. A cornerstone of this vision is its self-sufficient governance and funding model, which eliminates the need for external funding. At the heart of Cardano's innovative approach is an on-chain treasury system, consistently funded through transaction fees and treasury taxes. These mechanisms are pivotal for the platform's ongoing development. They are detailed in discussions with [Bingsheng Zhang](https://www.youtube.com/watch?v=Hyh3h_yX-S0), providing valuable insights into the treasury's operations.
```

Comment:
> and treasury taxes

what does this mean?

Reply:
I'm here referring to the `τ` parameter, the treasury growth rate. 
Thinking about it I should probably rephrase this sentence. Here is my suggestions:

> At the heart of Cardano's innovative approach is an on-chain treasury system, consistently funded each epoch by a fixed percentage of the virtual pot, which encompasses all transaction fees collected within that epoch and the ada allocated from the reserve, with the reserve's ada allocation determined by the monetary expansion rate.

Reply:
This sounds good

The term `treasury taxes` was confusing for me.